packages/sources/src/components/ozon-content/ozon-content.content.ts

```diff
@@ -224,6 +224,35 @@ export const content = [
       </Inhoud>
     `,
   },
+  {
+    title: 'Figuur',
+    content: `
+      <Inhoud>
+        <Al>Alde Feanen behoort tot het Natura 2000-landschap 'Meren en Moerassen'.</Al>
+        <Figuur
+          eId="chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+          wId="gm1979_2__chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+        >
+          <Titel>Afbeelding Titel</Titel>
+          <Illustratie
+            naam="https://nep-knooppunt-test.viewer.dso.kadaster.nl/publiek/omgevingsdocumenten/api/presenteren/v7/afbeeldingen/gm1979/_akn_nl_act_gm1979_2021_omgevingsplandelfzijl_nld_2/image1.png"
+            alt="Afbeelding 1"
+          />
+          <Bijschrift>Bijschrift bij het figuur.</Bijschrift>
```

Comment:
Volgens de specificatie heeft `Bijschrift` een verplicht attribuut `locatie`, willen we daar nu wat mee doen?

Reply:
#1730